src/app.ts

```diff
@@ -146,4 +189,57 @@ export default (
       method: 'deleteSecret',
     });
   });
+
+  app.on("issues", async(context: any) => {
+    const issueComment = context.issue({
+           body: "Thanks for submitting onboarding request!",
+         });
+    
+    // try {
+    //     const data = await parse(context);
+    // } catch {
+    //     app.log.info('Issue was not created using Issue form template (the YAML ones)');
+    //     console.log("We in here but failed");
+    // }
+    
+    const data = { //test data until parse working
+      targetCluster: ["cluster"],
+      teamName: "name",
+      desiredProjectName: "namespace",
+      projectDescription: "description of project",
+      usersNeedingAccess: "users",
+      namespaceQuota: ["custom"],
+      customQuota: "custTest",
+      GPGKey: ""
+    };
+
+    var cluster:string;
+    if (data["targetCluster"][0] == "smaug" || data["targetCluster"][0] == "infra"){
+      cluster = "moc/" + data["targetCluster"];
+    } else{
+      cluster = "emea/" + data["targetCluster"];
+    };
+
+    var quota:string;
+    if (data["namespaceQuota"][0] == "custom"){
+      quota = data["customQuota"];
+    } else{
+      quota = data["namespaceQuota"][0];
+    }
+
+    const payloadDict = {
+      namespace: data["desiredProjectName"],
+      group: data["teamName"],
+      quota: quota,
+      cluster: cluster,
+      description: data["projectDescription"],
+      users: data["usersNeedingAccess"],
+      GPG: data["GPGKey"]
+    };
```

Comment:
Feel free to adjust the issue template keys, so you don't have to do these mappings here and the data already comes in the right format from the issue. The more you hardcode into the controller the more complex future maintenance and changes gets.